pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -345,20 +322,15 @@ func (og *operationGenerator) GenerateDetachVolumeFunc(
 		}
```

Comment:
Sorry for the late response -- I missed the notification. GenereateDetachVolumeFunc is being called from OperationExecutor's DetachVolume which is being called by reconciler and the error is being passed up... So any error in this function should be logged by the reconciler. However it's safer not to rely on the callers to log the error (OperationExecutor doesn't do that too...) and rather print it several times than miss it altogether. So I think @ianchakeres' solution is correct.